### ISSUE TITLE
`flutter doctor`/`devices` shows disconnected devices "iPhone is not connected. Xcode will continue when iPhone is connected. (code -13)"

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#105970]
These errors can show up when the device was set up for wireless pairing.  It has also been reported for other disconnected devices that were never set up with wireless pairing but I'm not sure why that happens.
```
[✓] Connected device (3 available)
    • iPhone 11 (mobile) • CB23A1C8-7131-457E-9B14-6721BC996C3D • ios            • com.apple.CoreSimulator.SimRuntime.iOS-16-0 (simulator)
    • macOS (desktop)    • macos                                • darwin-x64     • macOS 12.4 21F79 darwin-x64
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 102.0.5005.61
    ! Error: iPhone is not connected. Xcode will continue when iPhone is connected. (code -13)
    ! Error: iPad is not connected. Xcode will continue when iPad is connected. (code -13)
```
Filter it out.

**Comment by jmagman** [comment]
I don't think we can rely on just the code -13 since it's used in other cases, so check that with part of the error string.

**Comment by christopherfujino** [comment]
@jmagman I'm guessing this was fixed by #105971 ?

**Comment by jmagman** [comment]
Yes, closing.
